src/app/Main/Body/QueryPage/Results/TableResults/TableResults-Options.component.tsx

```diff
@@ -73,22 +61,17 @@ const BlueRadio = withStyles(() =>
  * @returns {JSX.Element} - option bar component
  */
 export function ResultsTableOptionsComponent(props: {
-    isSumRowCol: { rowSum: boolean; colSum: boolean };
+    isSumRowCol: { showRowSum: boolean; showColSum: boolean };
     tableOption: DisplayOptionType;
     onDisplayOptionsChange: (displayOption: string) => void;
-    onShowRowSumToggle: (showSums: boolean) => void;
-    onShowColSumToggle: (showSums: boolean) => void;
+    onShowRowColSum: (name: string, checked: boolean) => void;

```

Comment:
Better would be something like: ``onSumOptionsChange(showRowSum: boolean, showColSum: boolean)``. By using ``name: string`` you loose type safety.
If you like you can also introduce an interface:
```
interface SumOptions {
  showRowSum: boolean,
  showColSum: boolean
}
```
and use it for the input and the output event.

Reply:
(name: string, checked: boolean) is defined by the return type of the material-ui checkbox event.

Reply:
Done, I think... Commit: "Use SumOptions-interface"